src/GraphQL/ObjectExtensions.cs

```diff
@@ -306,5 +306,44 @@ public static bool IsDefinedEnumValue(Type type, object value)
 
             return false;
         }
+
+        /*
+        public static IDictionary<string, object> ToObjectDictionary(this IDictionary<string, ArgumentValue> source)
+        {
+            return new ObjectDictionary(source);
+        }
+
+        private class ObjectDictionary : IDictionary<string, object>
```

Comment:
What is it?

Reply:
Backwards compatibility - if we want to provide a method to retrieve an `IDictionary<string, object>` that contains the argument values (like 3.x did) rather than `ArgumentValue` values.  If there is no reason to include this code, I'll remove it entirely.  What do you think?

Reply:
`IResolveFieldContext.Arguments` ?

Reply:
`IResolveFieldContext.Arguments` type should be changed to `IDictionary<string, ArgumentValue>`

Reply:
So then I'll delete this code.